backend/dashboard_metrics/models.py

```
@@ -0,0 +1,354 @@
+        constraints = [
+            models.UniqueConstraint(
+                fields=[
+                    "organization",
+                    "timestamp",
+                    "metric_name",
+                    "project",
+                    "tag",
+                ],
+                name="unique_hourly_metric",
+            )
+        ]
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)